src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
```

Comment:
it task description there is:
All instances of the `Animal` class must have properties health and a name. Health is 100 by default.

I think it means: health is 100 by default, but can be different
please make constructor take also health and set it to 100 if it is not passed

Reply:
Good point. I have changed this.
I just don't understand why should I add reduceHealth and die methods to Animal if it's only needed in herbivore. I have added this, but I think that now code is more complex. Could you explain it to me a little bit more?

Reply:
I know that I could add Animal.live outside the class, but the task was:
All alive animals should be in the static `Animal.alive` array. I think that word 'static' says that it should be added inside the class.